sdk/typescript/src/rpc/faucet-client.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import fetch from 'cross-fetch';
+
+import { FaucetResponse, SuiAddress } from '../types';
+import { HttpHeaders } from './client';
+
+export async function requestSuiFromFaucet(
+  endpoint: string,
+  recipient: SuiAddress,
+  httpHeaders?: HttpHeaders
+): Promise<FaucetResponse> {
+  const options = {
```

Comment:
FWIW if you avoid this options object and just define this inline on `fetch`, the type errors are way more obvious.